content/docs/09.administrator-guide/02.deployment/04.restack.md

```diff
@@ -1,56 +1,190 @@
+![](https://cdn.sanity.io/images/ev3amoz3/production/4c3c9385fd468fd49fcade0b405f089e14303bba-4320x2280.png)
 
-Make sure that this account has *AdministratorAccess*. This is how Restack can ensure an end-to-end cluster creation and cluster management process.
+Create an IAM Role by following these steps:
 
-1. Navigate to **Clusters** in the left-hand navigation menu.
-2. Select the **Credentials** tab.
-3. Click **Add credential**.
-    ![add-credentials](/docs/administrator-guide/deployment/restack/restack-add-credentials.png)
-4. Give a suitable title to your credentials for managing them later.
-5. Enter your [**AWS Access Key ID** and **AWS Secret Access key**.](https://docs.aws.amazon.com/accounts/latest/reference/root-user-access-key.html)
-6. Click **Add credential**.
-    ![add-aws-credentials](/docs/administrator-guide/deployment/restack/restack-add-AWS-creds.png)
+1. Log into your AWS Management Console as a user with administrator privileges and go to the IAM console.
 
->[How to get your AWS Access key ID and AWS Secret Access Key](https://docs.aws.amazon.com/accounts/latest/reference/root-user-access-key.html)
+2. Click the Roles tab and click on Create Role in the IAM sidebar.
 
-## One-click cluster creation with Restack
+ ![](https://cdn.sanity.io/images/ev3amoz3/production/7d1ba16ba5610e8947ca20d8746ee8424a9d4621-2000x1062.png)
+
+3. In Select type of trusted entity, click the Custom trust policy
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/666cb5015b3c5a29e6eef6216bdd97ef67297dc8-1770x1110.png)
+
+4. Copy and Paste into the custom policy field the following. You also can copy it directly from the Restack user interface modal.
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": [
+                    "arn:aws:iam::223746943638:user/connect",
+                    "arn:aws:iam::223746943638:role/connect"
+                ]
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "sts:ExternalId": "restack"
+                }
+            }
+        }
+    ]
+  }
+```
+
+5. Click the Next button and Add AdministratorAccess Permissions.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/2685ee99b153f46b38f99607407dfba72c5c6522-2000x1160.png)
+
+6. Click the Next button and Add Role Name `restack`, Review, and Create button.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/fb14345d290077195b986cb3931c0d39689dca6f-2000x1255.png)
+
+7. From IAM > Role overview, navigate to the role you just created and copy the ARN from there.
+
+![](https://cdn.sanity.io/images/ev3amoz3/production/e3750c4fb52873a781e328e81c70237d1968d5da-2560x1440.png)
+
+8. Copy the ARN to Restack's user interface and check by clicking on 'Connect cloud' if Restack can successfully assume the role in your AWS account.
```

Comment:
```suggestion
8. Paste the Role ARN to Restack's user interface and validate the setup by clicking on the 'Connect cloud' button. This will validate if Restack can successfully assume the role in your AWS account.
```